picard/tagger.py

```diff
@@ -185,7 +185,7 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
             self.set_log_level(logging.DEBUG)
 
         # Default thread pool
-        self.thread_pool = ThreadPoolExecutor()
+        self.thread_pool = ThreadPoolExecutor(max_workers=max(1, os.cpu_count()-2))
```

Comment:
Why this change? Why this value? It deserves a comment IMHO.

According to https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor

> Default value of max_workers is changed to min(32, os.cpu_count() + 4). This default value preserves at least 5 workers for I/O bound tasks. It utilizes at most 32 CPU cores for CPU bound tasks which release the GIL. And it avoids using very large resources implicitly on many-core machines.

Reply:
I can leave it as is, but when a bunch of Essentia extractors are running, it drastically slows down the entire computer, not only Picard. Maybe switch to a different thread pool?

Reply:
I see no problem in changing it, but please add a comment explaining how the new value was chosen

Reply:
Ended up removing that. Not really sure if we will have any problem in the real world, as extraction is currently waiting the API checking for duplicates before extracting anything. Kinda slows down the entire process, but also doesn't hog the computer extracting unnecessary stuff nor updates unnecessary data.